stats/opentelemetry/opentelemetry.go

```diff
@@ -284,14 +286,11 @@ func createFloat64Histogram(setOfMetrics map[MetricName]bool, metricName MetricN
 // Users of this component should use these bucket boundaries as part of their
 // SDK MeterProvider passed in. This component sends this as "advice" to the
 // API, which works, however this stability is not guaranteed, so for safety the
-// SDK Meter Provider should set these bounds.
+// SDK Meter Provider provided should set these bounds for corresponding
+// metrics.
 var (
-	// DefaultLatencyBounds are the default bounds for latency metrics. Users of
-	// this component should set these bucket boundaries as part of their SDK
-	// MeterProvider passed in for desired latency metrics.
+	// DefaultLatencyBounds are the default bounds for latency metrics.
 	DefaultLatencyBounds = []float64{0, 0.00001, 0.00005, 0.0001, 0.0003, 0.0006, 0.0008, 0.001, 0.002, 0.003, 0.004, 0.005, 0.006, 0.008, 0.01, 0.013, 0.016, 0.02, 0.025, 0.03, 0.04, 0.05, 0.065, 0.08, 0.1, 0.13, 0.16, 0.2, 0.25, 0.3, 0.4, 0.5, 0.65, 0.8, 1, 2, 5, 10, 20, 50, 100} // provide "advice" through API, SDK should set this too
```

Comment:
It would also be nice to show users how to do this, too.

Reply:
Added as part of example. Verified it worked by playing around with example configuration wrt bounds and seeing the result.